variables.tf

```diff
@@ -24,6 +24,24 @@ variable "label_prefix" {
   default     = "none"
 }
 
+variable "ig_rt_display_name" {
```

Comment:
There are already variables for the display names e.g. `nat_gateway_display_name`. Please use those instead of defining new ones.

Reply:
@hyder -This is fixed